src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp

```diff
@@ -100,11 +98,11 @@ namespace OpenMS
 
     // center main window
     setGeometry(
-      (int)(0.1 * QApplication::desktop()->width()),
-      (int)(0.1 * QApplication::desktop()->height()),
-      (int)(0.8 * QApplication::desktop()->width()),
-      (int)(0.8 * QApplication::desktop()->height())
-      );
+      (int)(0.1 * QGuiApplication::primaryScreen()->geometry().width()),
```

Comment:
think I messed around with primaryScreen in the past but there was an issue I can't recall. Maybe keep our eyes open (multi-monitor setups etc.) for bugs